mountpoint-s3-client/tests/endpoint.rs

```diff
@@ -88,3 +88,88 @@ async fn test_addressing_style_uri_fips_dualstack(addressing_style: AddressingSt
     })
     .await;
 }
+
+// Transfer acceleration can only be supported with virtual-hosted-style addressing
+#[test_case(AddressingStyle::Virtual)]
+#[tokio::test]
+async fn test_addressing_style_uri_transfer_acceleration(addressing_style: AddressingStyle) {
+    run_test(|_region| {
+        let uri = "https://s3-accelerate.amazonaws.com".to_string();
+        Endpoint::from_uri(&uri, addressing_style).unwrap()
+    })
+    .await;
+}
+
+async fn run_access_points<F: FnOnce(&str) -> Endpoint>(f: F) {
+    let sdk_client = get_test_sdk_client().await;
+    let (access_point, prefix) =
+        get_test_access_point_alias_and_prefix("test_access_point", AccessPointType::SingleRegion);
+
+    // Create one object named "hello"
+    let key = format!("{prefix}/hello");
+    let body = b"hello world!";
+    sdk_client
+        .put_object()
+        .bucket(&access_point)
+        .key(&key)
+        .body(ByteStream::from(Bytes::from_static(body)))
+        .send()
+        .await
+        .unwrap();
+
+    let region = get_test_region();
+    let endpoint = f(&region);
+    let config = S3ClientConfig::new().endpoint(endpoint);
+    let client = S3CrtClient::new(&region, config).expect("could not create test client");
+
+    let result = client
+        .get_object(&access_point, &key, None, None)
+        .await
+        .expect("get_object should succeed");
+    check_get_result(result, None, &body[..]).await;
+}
+
+#[test_case(AddressingStyle::Automatic)]
+#[test_case(AddressingStyle::Virtual)]
+#[test_case(AddressingStyle::Path)]
+#[tokio::test]
+async fn test_single_region_access_point_alias(addressing_style: AddressingStyle) {
+    run_access_points(|region| Endpoint::from_region(region, addressing_style).unwrap()).await;
+}
+
+async fn run_other_access_points<F: FnOnce(&str) -> Endpoint>(f: F, access_point_type: AccessPointType) {
```

Comment:
Why is the actual test different from the one in `run_access_points` (besides `access_point_type`)?

Reply:
Because, for Object Labda Access Point, Lambda function needs to be configured to add API support for Put Object. For multi region access points, Rust SDK is not supported. I have added comments for it.

Reply:
typo: "Object Lambda"

Reply:
maybe rename to `run_list_objects_test`, which is what this method does. The comment explains why.